### DISCUSSION TITLE
supertag not building on macOS Catalina

### DISCUSSION THREAD

**Discussion by miloder** [amoffat/supertag#26] [Q&A]
Hi there! It appears that supertag doesn't build under macOS Catalina because of fuse-sys. Any idea how we could fix this?

`(base) ********@******** supertag % cargo build  
   Compiling config v0.10.1
   Compiling fuse-sys v0.1.0 (/********/supertag/fuse-sys)
error: failed to run custom build command for `fuse-sys v0.1.0 (/********/supertag/fuse-sys)`

Caused by:
  process didn't exit successfully: `/********/supertag/target/debug/build/fuse-sys-920aad06fd96d7ac/build-script-build` (exit code: 101)
  --- stdout
  cargo:rerun-if-env-changed=OSXFUSE_NO_PKG_CONFIG
  cargo:rerun-if-env-changed=PKG_CONFIG
  cargo:rerun-if-env-changed=OSXFUSE_STATIC
  cargo:rerun-if-env-changed=OSXFUSE_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_STATIC
  cargo:rerun-if-env-changed=PKG_CONFIG_ALL_DYNAMIC
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_PATH
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_LIBDIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64-apple-darwin
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR_x86_64_apple_darwin
  cargo:rerun-if-env-changed=HOST_PKG_CONFIG_SYSROOT_DIR
  cargo:rerun-if-env-changed=PKG_CONFIG_SYSROOT_DIR

  --- stderr
  thread 'main' panicked at 'Invalid version of osxfuse: Failure { command: "\"pkg-config\" \"--libs\" \"--cflags\" \"osxfuse\" \"osxfuse >= 2.6.0\"", output: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "Package osxfuse was not found in the pkg-config search path.\nPerhaps you should add the directory containing `osxfuse.pc\'\nto the PKG_CONFIG_PATH environment variable\nNo package \'osxfuse\' found\nPackage osxfuse was not found in the pkg-config search path.\nPerhaps you should add the directory containing `osxfuse.pc\'\nto the PKG_CONFIG_PATH environment variable\nNo package \'osxfuse\' found\n" } }', fuse-sys/build.rs:36:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed`

**Comment by amoffat** [comment]
Do you have osxfuse installed on your system? If so, it might be related to this issue https://github.com/amoffat/supertag/issues/18
